src/Seq.App.EmailPlus/EmailApp.cs

```diff
@@ -132,18 +150,18 @@ public void On(Event<LogEventData> evt)
 
             var to = FormatTemplate(_toAddressesTemplate.Value, evt, base.Host);
             var body = FormatTemplate(_bodyTemplate.Value, evt, base.Host);
-            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "").Replace("\n", "");
+            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "")
+                .Replace("\n", "");
             if (subject.Length > MaxSubjectLength)
                 subject = subject.Substring(0, MaxSubjectLength);
 
-            var client = new SmtpClient(Host, Port ?? 25) {EnableSsl = EnableSsl ?? false};
-            if (!string.IsNullOrWhiteSpace(Username))
-                client.Credentials = new NetworkCredential(Username, Password);
+            var result = await _mailGateway.Send(_options, new MimeMessage(new List<InternetAddress> {MailboxAddress.Parse(From)},
+                new List<InternetAddress> {MailboxAddress.Parse(to)}, subject,
+                (new BodyBuilder() {HtmlBody = body}).ToMessageBody()));
+
+            if (!result.Success)
+                throw result.Errors;
```

Comment:
This will cause the loss of the original stack trace; I might head towards making `Send()` throw, but I'll merge now to avoid more churn on this long and storied PR :-)

Reply:
No worries. I do have handling for this in further commits - will send a new PR for the next set.